test/src/e2e_vm_tests/test_programs/should_pass/language/b256_literals/src/main.sw

```diff
@@ -0,0 +1,19 @@
+script;
+
```

Comment:
One more thing is that this test should have more diverse sets of values to make sure that things like endianess works correctly.